app/controllers/assets_controller.rb

```diff
@@ -4,6 +4,8 @@ def tiles
       image = AssetGenerator.protected_area_tile(protected_area)
     elsif params[:type] == "country"
       image = AssetGenerator.country_tile(country)
+    elsif params[:type] == "region"
+      image = AssetGenerator.region_tile(region)
```

Comment:
Would this be a good situation to use a case..when statement instead of if..else? From a readability perspective, I don't think it makes any difference performance-wise

Reply:
Thanks. I've refactored a bit differently now.
I didn't use a case statement in the end as I found I could make use of the fact that method names and area types name are similar.

Reply:
Nicely done 🔥